transport/handler/release.go

```diff
@@ -45,8 +45,15 @@ func (h *Handler) getRelease(w http.ResponseWriter, r *http.Request) {
 }
 
 func (h *Handler) deleteRelease(w http.ResponseWriter, r *http.Request) {
+	var input model.DeleteReleaseInput
+	if err := util.UnmarshalRequest(r, &input); err != nil {
+		util.WriteResponseError(w, resperrors.NewBadRequestError().Wrap(err).WithMessage(err.Error()))
+		return
+	}
+
 	if err := h.ReleaseSvc.Delete(
 		r.Context(),
+		model.ToSvcDeleteReleaseInput(input),
```

Comment:
It might feel like a lot of unnecessary boilerplate code for an input with just one field, but I think it is better to do it this way to be consistent with the rest of the application. Besides, you let GitHub Copilot generate these simple function anyway, so it only takes a minute of work.